<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeToCLog</class>
 <widget class="QDialog" name="MergeToCLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merge ot Clog file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Merge from:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="filesToExport">
        <property name="toolTip">
         <string>export files path</string>
        </property>
        <property name="whatsThis">
         <string>merge files path</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getFiles">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>19</height>
         </size>
        </property>
        <property name="toolTip">
         <string>select files to export</string>
        </property>
        <property name="whatsThis">
         <string>select files to merge</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Save to:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="ExportTo">
        <property name="toolTip">
         <string>exports path</string>
        </property>
        <property name="whatsThis">
         <string>save path</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getDir">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>19</height>
         </size>
        </property>
        <property name="toolTip">
         <string>select exports path</string>
        </property>
        <property name="whatsThis">
         <string>select save path</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>132</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MergeToCLog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MergeToCLog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
